### ISSUE TITLE
Question about reproducing LOL-v1 training results and training settings

### ISSUE THREAD

**Issue by OceanSong2003** [98Hao/MEGF-Net#1]
Hi, thank you for releasing the code and pretrained weights for MEGF-Net.
I am trying to reproduce the training results from scratch and encountered a noticeable gap on LOL-v1.

My environment is:

PyTorch: 2.1.1+cu118
Dataset: LOL-v1, 485 training pairs / 15 test pairs
Crop size: 256
Batch size: 2
Optimizer: Adam
Scheduler milestones: 250 / 350 / 450 / 475, gamma = 0.5

I first tested the released LOLv1.pkl checkpoint and obtained:

PSNR: 24.8330
SSIM: 0.8595

which is consistent with the paper.

However, when training from scratch using the released training code, my best LOL-v1 result is:

Best PSNR: 23.2125
SSIM: 0.8451
Best PSNR epoch: 303

At epoch 470, my model obtains:

PSNR: 22.4622
SSIM: 0.8450

I also inspected the released LOLv1.pkl checkpoint metadata:

epoch: 470
optimizer steps: 114210
initial_lr: 3e-4
current lr: 3.75e-5
milestones: 250, 350, 450, 475
batch size inferred from steps: 2

The number of optimizer steps exactly matches:

ceil(485 / 2) × 470 = 114210

so the number of training samples, batch size, epochs, and scheduler appear to match the released setup.

I also checked:

Low/High training pairs are correctly matched.
Data augmentation is synchronized between input and GT.
Evaluation uses RGB PSNR/SSIM.
The training-time validation result matches my independent evaluation.

Interestingly, LOL-v2 Synthetic can be reproduced quite closely. My model trained from scratch obtains:

PSNR: 24.8757
SSIM: 0.9337

while the released checkpoint gives approximately:

PSNR: 24.7939
SSIM: 0.9340

Therefore, the training pipeline itself seems to work correctly, and the reproduction issue appears mainly on LOL-v1.

I also noticed one inconsistency between the paper and the released code/checkpoint:

The paper states that the initial learning rate is 2e-3.
The released training code uses 3e-4.
The optimizer state stored in the official LOLv1.pkl checkpoint also shows initial_lr = 3e-4.

Could you please clarify:

Was the released LOL-v1 checkpoint trained exactly with the current GitHub training code?
Was any pretrained model, fine-tuning stage, different random seed, or additional training strategy used for LOL-v1?
Should the initial learning rate for the reported results be 2e-3 or 3e-4?
Is there a specific random seed or LOL-v1 dataset version that should be used to reproduce the reported result?

I can provide my training logs, TensorBoard records, checkpoints, and dataset hashes if helpful.

Thank you very much!
